files/deployment.yaml

```diff
@@ -49,12 +49,17 @@
       register: kubeconfig_token
       become: "{{ as_root }}"
 
+    - name: oc get server
+      command: oc whoami --show-server=true
+      register: oc_server
+      become: "{{ as_root }}"
+
     - name: Create dev.yml
       copy:
         content: |
           # See https://github.com/packit/deployment/blob/master/vars/dev_template.yml
           project: myproject
-          host: https://127.0.0.1:8443
+          host: {{ oc_server.stdout }}
```

Comment:
Uh, thinking about it more, this might be a disaster if you happen to forget to spin up a local cluster before running `make check_in_container` - in which case you might still be logged to a prod/stg cluster.

Reply:
Indeed it would :slightly_smiling_face: 

Would it be possible to try to use the `minishift` command first (`minishift openshift` should have some way to get this value) and if that does not work for some reason, fall back to localhost?

Reply:
Yup, `minishift ip` is the way. I changed the code, the current version doesn't work, but I'll fix it once I have my proper internet connection back.

Reply:
changed, PTAL

Reply:
solved